### ISSUE TITLE
amp-sidebar not allowing scrolling right to the bottom on Nexus 5

### ISSUE THREAD

**Issue by matthewbrightlabs** [ampproject/amp-by-example#888]
Hi there,

I have been implementing AMP for a client's homepage. One of my tasks was upgrading to an amp-sidebar. It has two levels of accordion i.e.

List item 1
 - Child 1
   -- Grandchild 1
   -- Grandchild 2
 - Child 2
List item 2
List item 3

The page validates successfully with my amp markup. 
- Nexus 5 Google Chrome, there appears to be 30-40px cutoff the bottom of the sidebar
- iPhone 6 Google Chrome allows scrolling right to the bottom
- I have not edited any height css of the container or amp-sidebar element.

I tested the https://ampbyexample.com/componenets/amp-sidebar and it does scroll correctly right to the bottom with the lists fully expanded. I noticed the example only uses one level of accordion.

**Comment by insanesvk** [comment]
yeah, this bugs me as well, sidebar injects this to the bottom of the sidebar:
`<div style="height: 10vh; width: 100%; background-color: transparent;"></div>`

It would be really neat if this would be configurable.

**Comment by sebastianbenz** [comment]
Please file a feature request on ampproject/amphtml.
